sql/updates/post-0.4.1--0.4.2-dev.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TRIGGER trigger_main_on_change_chunk_constraint
```

Comment:
This adds a trigger while we decided to remove them.

Reply:
Fixed (In PR #198)